src/Riak/Object.php

```diff
@@ -67,4 +74,63 @@ public function getContentType()
     {
         return $this->getHeader('Content-Type');
     }
+
+    public function getIndexes()
+    {
+        return $this->indexes;
+    }
+
+    public function getIndex($indexName)
+    {
+        return $this->indexes[$indexName];
```

Comment:
Whenever working with an array, you want to check if a value is set before returning it or it will throw an undefined error. You could rewrite this as:

```return isset($this->indexes[$indexName]) ? $this->indexes[$indexName] : null;```